apps/web/src/pages/quick-start/steps/DigestPreview.tsx

```diff
@@ -28,8 +27,7 @@ export function DigestPreview() {
     <GetStartedLayout
       header={
         <>
-          <HeaderTitle>Set-up steps to get started</HeaderTitle>
-          <HeaderSecondaryTitle>Quick Start Guide</HeaderSecondaryTitle>
+          <Title>Get started</Title>
```

Comment:
Are we sure the titles need to be removed?

Reply:
Right, it was not supposed to be, reverting it.

Reply:
done
<img width="1511" alt="image" src="https://github.com/novuhq/novu/assets/29251776/65a9dc6a-f819-4654-b872-d0df8d387919">


Step 1 stays with the new design:
<img width="1511" alt="image" src="https://github.com/novuhq/novu/assets/29251776/cbd786ca-cc1b-4f7a-91f0-9d5992c69595">